iib/workers/tasks/build.py

```diff
@@ -340,21 +350,22 @@ def _get_present_bundles(from_index, base_dir):
         return [], []
 
     # Transform returned data to parsable json
-    unique_present_bundles = []
+    unique_present_bundles: List[BundleImage] = []
     unique_present_bundles_pull_spec = []
     present_bundles = get_bundle_json(bundles)
-
     for bundle in present_bundles:
         bundle_path = bundle['bundlePath']
         if bundle_path in unique_present_bundles_pull_spec:
             continue
         unique_present_bundles.append(bundle)
         unique_present_bundles_pull_spec.append(bundle_path)
-
     return unique_present_bundles, unique_present_bundles_pull_spec
 
 
-def _get_missing_bundles(present_bundles, bundles):
+def _get_missing_bundles(
+    present_bundles: List[BundleImage],
+    resolved_bundles: List[str],
```

Comment:
Since the variable name changed, you probably need to change the docstring here too :)

Reply:
Thanks, I forgot to do that.